src/io/flutter/view/DiagnosticsTreeCellRenderer.java

```diff
@@ -217,6 +209,9 @@ else if (!node.getDescription().isEmpty()) {
         appendText(node.getDescription(), textAttributes);
       }
 
+      // TODO(devoncarew): For widgets that are definied in the current project, we could consider
```

Comment:
I like this idea. The relative path perhaps shown in grey italics could provide some good context.
This is the sort of feature we should put behind a checkbox in the settings menu to get user feedback and provide an escape hatch if users find it annoying.